src/servers/ZoneServer2016/zonepackethandlers.ts

```diff
@@ -1744,11 +1830,161 @@ export class ZonePacketHandlers {
         }
       } else {
         // to external container
-        // not used for now with the external container workaround
+        const sourceContainer = client.character.getItemContainer(itemGuid),
+          targetCharacter = client.character.mountedContainer;
+
+        if (
+          !targetCharacter ||
+          !(targetCharacter instanceof BaseLootableEntity) ||
+          !isPosInRadius(
+            targetCharacter.interactionDistance,
+            client.character.state.position,
+            targetCharacter.state.position
+          )
+        ) {
+          server.sendChatText(client, "Invalid target character 2!");
+          return;
+        }
+
+        const targetContainer = targetCharacter.getContainer();
+        if (!targetContainer) {
+          server.sendChatText(client, "Invalid target container 2!");
+          return;
+        }
+
+        const loadoutItem = client.character.getLoadoutItem(itemGuid);
+        if (loadoutItem) {
+          client.character.transferItemFromLoadout(
+            server,
+            targetContainer,
+            loadoutItem
+          );
+          client.character.mountContainer(server, targetCharacter);
+          return;
+        }
+
+        if (!sourceContainer) {
+          server.sendChatText(client, "Invalid source container 2!");
+          return;
+        }
+
+        if (containerGuid == "0xffffffffffffffff") {
```

Comment:
Same could be a const, so it's more understandable.

Reply:
done 👍